### PR TITLE
added function paddle.tensor.manipulation and test draft For moveaxis

### PR DESCRIPTION
**Pull request by Shivam7-1**
Issue link: https://github.com/unifyai/ivy/issues/19157 
Created `moveaxis `Function with Updated test draft for it

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by Shivam7-1** [comment]
Hii @juliagsy  Could you please review my PR

File: ivy/functional/frontends/paddle/tensor/manipulation.py [hunk 1]
```diff
@@ -155,6 +155,17 @@ def take_along_axis(arr, indices, axis):
     return ivy.take_along_axis(arr, indices, axis)
 
 
+@with_supported_dtypes(
```
Comment (juliagsy) [inline ivy/functional/frontends/paddle/tensor/manipulation.py:158]: Hey! Thanks for the great work! might be some error caused when resolving merge conflicts, but the formatting and decorators are incorrect at the moment for both functional and tests 😅 could you please fix them? Thanks!

Comment (Shivam7-1) [reply ivy/functional/frontends/paddle/tensor/manipulation.py:162]: @juliagsy yes I got the error and  done 👍 Changes Can you Review it again

**Comment by Shivam7-1** [comment]
@juliagsy done changes  👍

**Comment by Shivam7-1** [comment]
Hii @juliagsy  Can you please Review my PR

File: ivy/functional/frontends/paddle/tensor/manipulation.py [hunk 2]
```diff
@@ -171,5 +186,6 @@ def take_along_axis(arr, indices, axis):
     "paddle",
 )
 @to_ivy_arrays_and_back
-def rot90(x, k=1, axes=(0, 1), name=None):
```
Comment (juliagsy) [inline ivy/functional/frontends/paddle/tensor/manipulation.py:174]: I think this should be kept, the duplication above should be removed

File: ivy/functional/frontends/paddle/tensor/manipulation.py [hunk 3]
```diff
@@ -171,5 +186,6 @@ def take_along_axis(arr, indices, axis):
...
-    return ivy.rot90(x, k=k, axes=axes)
+def moveaxis(x, source, destination, name=None):
```
Comment (juliagsy) [inline ivy/functional/frontends/paddle/tensor/manipulation.py:189]: you'll need to add the unsupported dtype decorator etc for this function as well

File: ivy/functional/frontends/paddle/tensor/manipulation.py [hunk 4]
```diff
@@ -155,6 +155,21 @@ def take_along_axis(arr, indices, axis):
...
+        "2.5.1 and below": (
+            "bool",
+            "float32",
+            "float64",
+            "int32",
+            "int64",
+        )
+    }
+)
 @with_supported_device_and_dtypes(
```
Comment (juliagsy) [inline ivy/functional/frontends/paddle/tensor/manipulation.py:173]: there's 2 decorators here, please keep the original and remove the duplicate

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py [hunk 5]
```diff
@@ -693,21 +738,34 @@ def test_paddle_take_along_axis(
 def test_paddle_rot90(
     *,
     dtype_m_k_axes,
+
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py:741]: please revert this

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py [hunk 6]
```diff
@@ -693,21 +738,34 @@ def test_paddle_take_along_axis(
...
     on_device,
     fn_tree,
     frontend,
     backend_fw,
     test_flags,
 ):
+
+    input_dtypes, x, source_axis, destination_axis = dt_x_source_destination
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py:749]: please remove this, it's a duplication

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py [hunk 7]
```diff
@@ -693,21 +738,34 @@ def test_paddle_take_along_axis(
...
+    helpers.test_frontend_function(
+        input_dtypes=input_dtypes,
+
     input_dtype, m, k, axes = dtype_m_k_axes
     helpers.test_frontend_function(
         input_dtypes=input_dtype,
+
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py:756]: please revert this

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py [hunk 8]
```diff
@@ -693,21 +738,34 @@ def test_paddle_take_along_axis(
...
         backend_to_test=backend_fw,
         frontend=frontend,
         test_flags=test_flags,
         fn_tree=fn_tree,
         on_device=on_device,
+
+        x=x[0],
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py:763]: this is a duplication as well

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hey! There's still errors unfortunately, I've left some comments to help in cleaning up, thanks!

**Comment by juliagsy** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**